docs/core/tools/global-tools.md

```diff
@@ -1,135 +1,270 @@
 ---
-title: .NET Core Global Tools
-description: An overview of what .NET Core Global Tools are and the .NET Core CLI commands available for them. 
+title: .NET Core tools
+description: How to install, use, update, and remove .NET Core tools. Covers global tools, tool-path tools, and local tools. 
 author: KathleenDollard
-ms.date: 05/29/2018
+ms.date: 02/12/2020
 ---
-# .NET Core Global Tools overview
+# How to manage .NET Core tools
 
-[!INCLUDE [topic-appliesto-net-core-21plus.md](../../../includes/topic-appliesto-net-core-21plus.md)]
+**This article applies to:** ✔️ .NET Core 2.1 SDK and later versions
 
-A .NET Core Global Tool is a special NuGet package that contains a console application. A Global Tool can be installed on your machine on a default location that is included in the PATH environment variable or on a custom location.
+A .NET Core tool is a special NuGet package that contains a console application. A tool can be installed on your machine in the following ways:
 
-If you want to use a .NET Core Global Tool:
+* As a global tool.
 
-* Find information about the tool (usually a website or GitHub page).
-* Check the author and statistics in the home for the feed (usually NuGet.org).
-* Install the tool.
-* Call the tool.
-* Update the tool.
-* Uninstall the tool.
+  The tool binaries are installed in a default directory that is added to the PATH environment variable. You can invoke the tool from any directory on the machine without specifying its location. One version of a tool is used for all directories on the machine.
+
+* As a global tool in a custom location (also known as a tool-path tool).
+
+  The tool binaries are installed in a location that you specify. You can invoke the tool from the installation directory or by providing the directory with the command name or by adding the directory to the PATH environment variable. One version of a tool is used for all directories on the machine.
+
+* As a local tool (applies to .NET Core SDK 3.0 and later).
+
+  The tool binaries are installed in a default directory. You invoke the tool from the installation directory or any of its subdirectories. Different directories can use different versions of the same tool.
+  
+  The .NET CLI uses manifest files to keep track of which tools are installed as local to a directory. These manifest files are saved in the source code repository, so they're available when a contributor checks out the code from the repository. The contributor can then invoke a single .NET Core CLI command that installs all of the tools that are listed in the manifest files.
```

Comment:
Changing this. It made it sound like this was the only scenario you would use this, but it may not be.

```suggestion
  The .NET CLI uses manifest files to keep track of which tools are installed as local to a directory. When the manifest file is saved in the root directory of a source code repository, a contributor can clone the repository and invoke a single .NET Core CLI command that installs all of the tools listed in the manifest files.
```